recipe/meta.yaml

```diff
@@ -1,45 +1,44 @@
 {% set name = "dm-tree" %}
-{% set version = "0.1.5" %}
+{% set version = "0.1.7" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dm-tree-{{ version }}.tar.gz
-  sha256: a951d2239111dfcc468071bc8ff792c7b1e3192cab5a3c94d33a8b2bda3127fa
-  patches:
-    - patches/0001-Contain-bazel-root.patch
-    - patches/0002-cxxopt-for-osx-sysroot.patch   # [osx]
+  sha256: 30fec8aca5b92823c0e796a2f33b875b4dccd470b57e91e6c542405c5f77fd2a
 
 build:
   number: 0
   skip: true   #[ppc64le]
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 
 # Need these up here for conda-smithy to handle them properly.
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - python
-    - bazel
-    - m2-patch  # [win]
-    - patch     # [not win]
+    - cmake
+    - make  # [not win]
+    - git
   host:
-    - pip
     - python
-    - six >=1.12.0 # needed because we can install by "python setup.py install" which has no --no-deps switch
+    - pip
+    - setuptools
+    - wheel
   run:
     - python
-    - six >=1.12.0
 
 test:
   imports:
     - tree
   commands:
+    - pip check
     - python -m unittest tree.tree_test tree.tree_benchmark
   requires:
+    - pip
     - attrs
```

Comment:
The remaining dependencies appear to require pinnings: https://github.com/deepmind/tree/blob/0.1.7/requirements-test.txt

Reply:
Good catch. Thanks! I've updated those.